docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md

```diff
@@ -0,0 +1,45 @@
+# 日志
+
+Nebula 使用 [glog](https://github.com/google/glog) 来打印日志，使用 gflag 控制日志的级别，并提供 HTTP 接口在运行时动态改变日志级别，以方便追踪问题。
+
+## 参数说明
+
+### glog 中的两个主要参数
+
+- minloglevel 0-4： 对应的日志级别分别为 INFO(DEBUG)，WARING，ERROR，FATAL。通常在调试环境设置为 0，生产环境设置为 1。
```

Comment:
minloglevel 0-3?  numbers of severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively

refer to  https://github.com/google/glog/blob/master/src/glog/log_severity.h

Reply:
@critical27 please help check

Reply:
It seems the minloglevel 0-4 is DEBUG, INFO, WARNING, ERROR, FALAL. And the `v` is valid when `minloglevel` is 0 (It used for debug by printing more detailed log).

Reply:
BTW, the doc here 0-4 list `INFO(DEBUG)，WARING，ERROR，FATAL` seems a little weird. WARNING is alos a typo.

Reply:
Thanks for your reply, please help update this. @whitewum